tasks/update.yml

```diff
@@ -1,13 +1,22 @@
 ---
-- name: Run database updates.
-  command: "{{ drush_path }} updatedb -y"
+
+- name: Detect Drupal version.
+  command: drush status drupal-version --format=list
   args:
     chdir: "{{ drupal_core_path }}"
+  register: drupal_core_version_output
+
+- name: Set Drupal version.
+  set_fact:
+    drupal_core_version: "{{ drupal_core_version_output.stdout }}"
 
-# TODO: Import configuration if configured?
-# TODO: Other commands if configured?
+- name: Run post-update tasks.
+  command: "{{ item.command }}"
+  args:
+    chdir: "{{ item.chdir | default(omit) }}"
+  with_items: "{{ drupal_deploy_update_tasks }}"
 
 - name: Rebuild Drupal caches.
-  command: "{{ drush_path }} cache-rebuild --quiet"
+  command: "{{ drush_path }} {{ (drupal_core_version >= '8') | ternary('cache-rebuild','cc all') }} --quiet"
```

Comment:
Guess the comparison should always work (?) but there's also the safer [`version_compare`](http://docs.ansible.com/ansible/latest/playbooks_tests.html#version-comparison) filter.